storage/innobase/btr/btr0scrub.cc

```diff
@@ -49,54 +50,57 @@ uint srv_background_scrub_data_interval = (7 * 24 * 60 * 60);
 * statistics for scrubbing by background threads
 */
 static btr_scrub_stat_t scrub_stat;
-static ib_mutex_t scrub_stat_mutex;
-mysql_pfs_key_t scrub_stat_mutex_key;
+static ib_mutex_t       scrub_stat_mutex;
+mysql_pfs_key_t		scrub_stat_mutex_key;
 
 static uint scrub_compression_level = page_zip_level;
 
-/**************************************************************//**
-Log a scrubbing failure */
-static
-void
+/** Log a scrubbing failure
+@param[in]	index
+@param[in,out]	scrub_data
```

Comment:
Lost comment on move. There are some more, I am only pointing out those which add something more than arg name (no point in having "param[in] block block" comments)
```suggestion
@param[in,out]	scrub_data data to store statistics on
```